remote_attestation/java/src/com/google/oak/remote_attestation/AttestationEngine.java

```diff
@@ -25,74 +25,66 @@
 import oak.remote_attestation.AttestationIdentity;
 import oak.remote_attestation.AttestationInfo;
 import oak.remote_attestation.AttestationReport;
```

Comment:
Pre-existing: does it makes sense to move the protos to `com.google.oak` package too? or perhaps to `com.google.oak.protos`?

Reply:
Thanks. Created #2205 to track this.